lib/client/bundler/default.js

```diff
@@ -99,122 +100,85 @@ module.exports = function(ss,client,options){
 
   /**
    * @ngdoc method
-   * @name bundler.default:default#assetLoader
+   * @name bundler.default:default#system
    * @methodOf bundler.default:default
    * @function
    * @description
-   * Return entry for the JS loader depending on the includes.system client config.
-   *
-   * @returns {AssetEntry} Loader resource
-   */
-  function assetLoader() {
-    return ss.bundler.systemLibs();
-  }
-
-  /**
-   * @ngdoc method
-   * @name bundler.default:default#systemModule
-   * @methodOf bundler.default:default
-   * @function
-   * @description
-   * Return the resource for a registered system module by the given name. It uses
-   * the default wrapCode for module registration with require.
-   *
-   * @param {String} name Logical Module Name
-   * @returns {AssetEntry} Module resource
-   */
-  function systemModule(name) {
-    switch(name) {
-      //case "eventemitter2":
-      //case "socketstream":
-      default:
-        return ss.bundler.systemModule(name)
-    }
-  }
-
-  /**
-   *
-   * @param path
-   * @param opts
-   * @param cb
-   * @returns {*}
-   */
-  function assetJS(path, opts, cb) {
-    return ss.bundler.loadFile(path, 'js', opts, function(output) {
-      //TODO with options compress saved to avoid double compression
-      output = ss.bundler.wrapCode(output, path, opts.pathPrefix);
-      if (opts.compress && path.indexOf('.min') === -1) {
-        output = ss.bundler.minifyJSFile(output, path);
-      }
-      return cb(output);
-    });
-  }
-
-  /**
+   * Return entries for the loader/start/module depending on includes.system client config and system assets.
+   * Multiple arguments will attempt to add module for the name.
+   * Special module names are 'loader' and 'start'.
    *
-   * @param path
-   * @param opts
-   * @param cb
-   * @returns {*}
+   * @returns {AssetEntry} Resource entries array
    */
-  function assetWorker(path, opts, cb) {
-    return ss.bundler.loadFile(path, 'js', opts, function(output) {
-      if (opts.compress) {
-        output = ss.bundler.minifyJSFile(output, path);
+  function systemModule(first_name) {
```

Comment:
Identifier 'first_name' is not in camel case.<br>'first_name' is defined but never used.